modules/govuk_jenkins/manifests/jobs/content_data_api.pp

```diff
@@ -16,7 +16,7 @@
 
   $check_name = 'etl-content-data-api'
   $service_description = 'Content Data API ETL [Extract - transform - load]'
-  $job_url = "https://deploy.${::aws_environment}.govuk.digital/job/content_data_api_import_etl_master_process/"
+  $job_url = "https://deploy.${app_domain}/job/content_data_api_import_etl_master_process/"
```

Comment:
I'm not sure if this is right, because the job exists in AWS in production and staging (for example: https://deploy.blue.production.govuk.digital/job/content_data_api_import_etl_master_process/).

I see what you mean about the redirect not working, but I think you've uncovered a problem for all our jobs in integration! For example https://deploy.integration.govuk.digital/job/smokey/ also redirects incorrectly, as does https://deploy.integration.govuk.digital/job/search-api-fetch-analytics-data/.

Ideally we would find a way to avoid Jenkins doing the redirect... but I have no idea how we could do that.

Reply:
Good point - I've removed this commit so it stays as is for now.